### PR TITLE
Add ingester downscaling docs

### PR DESCRIPTION
**Pull request by jhalterman**
#### What this PR does

Updates the Mimir docs to describe the new ingester downscaling procedures.

#### Which issue(s) this PR fixes or relates to

Followup to #8956.

#### Checklist

- [ ] Tests updated.
- [x] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

**Comment by pstibrany** [comment]
This is a duplicate of https://github.com/grafana/mimir/pull/9132, I was just waiting for @tacole02's review there.

File: docs/sources/mimir/manage/run-production-environment/scaling-out.md
Comment (pstibrany) [inline docs/sources/mimir/manage/run-production-environment/scaling-out.md:129]: This section is removed, but the documentation still says "Complete the following steps to scale down ingesters deployed in a single zone", and it's not quite clear how it translates to multiple zones. Procedure for multi-zone setup is the same, but we should mention that and perhaps remove that "single zone" sentence. WDYT?

Comment (jhalterman) [reply docs/sources/mimir/manage/run-production-environment/scaling-out.md:129]: I agree. I reworded that sentence a bit. Lmk what you think.

**Review by pstibrany — APPROVED** [review]
Thank you, this looks good from technical point of view.

**Comment by jhalterman** [comment]
Will wait for a review from @tacole02.

File: docs/sources/mimir/manage/run-production-environment/scaling-out.md
Comment (tacole02) [inline docs/sources/mimir/manage/run-production-environment/scaling-out.md:93]:
```suggestion
- The configured `-querier.query-store-after` setting
- Two times the configured `-blocks-storage.bucket-store.sync-interval` setting
- Two times the configured `-compactor.cleanup-interval` setting
```

File: docs/sources/mimir/manage/run-production-environment/scaling-out.md
Comment (tacole02) [inline docs/sources/mimir/manage/run-production-environment/scaling-out.md:99]:
```suggestion
   b. Wait until the API endpoint call has successfully returned and the ingester has logged "finished flushing and shipping TSDB blocks".
```

File: docs/sources/mimir/set-up/jsonnet/migrate-from-microservices-to-read-write.md
Comment (tacole02) [inline docs/sources/mimir/set-up/jsonnet/migrate-from-microservices-to-read-write.md:301]: Is `ingester-zone-a` a setting? If so, it would be ideal to specify, i.e. "in the `ingester-zone-a` setting."

**Review by tacole02 — APPROVED** [review]
This looks great!

File: docs/sources/mimir/set-up/jsonnet/migrate-from-microservices-to-read-write.md
Comment (jhalterman) [reply docs/sources/mimir/set-up/jsonnet/migrate-from-microservices-to-read-write.md:301]: It's not a Mimir setting exactly, but it is the standard name for the zone-A statefulset used in some of our deployment tooling/configuration such as in the Helm chart and also in the [rollout operator](https://github.com/grafana/rollout-operator).

**Pull request merged**
